app/Models/User.php

```diff
@@ -338,6 +338,33 @@ public function setApiKey($key)
         $this->api->save();
     }
 
+    /**
+     * Constants referring to the user profile extras (kudosu,
+     * recent activities, etc).
+     */
+    const USER_PAGE = 1;
```

Comment:
Also part of `UserProfileCustomization` and see how thing like this is handled in beatmap modes.